### PR TITLE
add dorafactory auction request

### PR DESCRIPTION
**Pull request by tinydjp**

### CODE REVIEW

File: crowdloans/projects/kusama.json [hunk 1]
```diff
@@ -338,6 +338,15 @@
...
-    }
+  },  
+  {
+    "paraid": "2115",
+    "name": "DORA KSM Parachain",
+    "token": "DORA",
+    "description": "DoraFactory Kusama parachain",
+    "overview": "Dora Factory is the DAO-as-a-Service infrastructure that empowers decentralized organizations and open source ventures. The DORA KSM Parachain is Dora Factory’s Kusama parachain, which supports pallets like quadratic funding and multisig wallet. In connection with DoraHacks, the Dora Factory KSM Parachain will be able to support open source communities, hackathons, quadratic funding grants, and DAOs in general from the Kusama ecosystem.",
+    "website": "https://dorafactory.org",
+    "icon": "https://dorafacory-assets.s3.ap-southeast-1.amazonaws.com/dorafactory-logo.svg",
```
Comment (crystalynliu) [inline crowdloans/projects/kusama.json:349]: could you add an icon in [our repo](https://github.com/parallel-finance/auction-toolkit/tree/main/icons/chains) and change the URL format to `https://auction-us-east-2.s3.us-east-2.amazonaws.com/icons/chains/<image-name>`?

**Comment by OhamwiParalell** [comment]
Parallel Finance team, please ensure the added rewards of 4 HKO per KSM contributed. Thank you

**Comment by i5possible** [comment]
Could you provide necessary info to create the vault?

**Comment by Helen8** [comment]
@tinydjp https://docs.parallel.fi/heiko-mainnet/crowdloan-v2-kusama-project-onboarding - pls see here on the information needed to onboard Dora. As soon as we get it, we can add.

File: crowdloans/projects/kusama.json (hunk 1 above)
Comment (tinydjp) [reply crowdloans/projects/kusama.json]: > @tinydjp https://docs.parallel.fi/heiko-mainnet/crowdloan-v2-kusama-project-onboarding - pls see here on the information needed to onboard Dora. As soon as we get it, we can add.

We've just updated the lacked information, please  help review again, thanks!

**Comment by tinydjp** [comment]
========Vault information==========
ParaId: 2115
Lease period: 20-27 
cToken: cKSM-11895000-11994000
EndBlock 

Auction Starts
March 22nd, 2022 - Block #11895000
Bidding Ends
March 29th, 2022 - Block #11994000
Slot Duration
Lease Period 20-27; April 6th, 2022 - March 15th, 2023

**Comment by crystalynliu** [comment]
Hi @tinydjp could you provide the endblock of the crowdloan?

**Comment by crystalynliu** [comment]
Hi @tinydjp I notice your team attend Auction 31st and the lease period is not the same as you provide, could you double check it?
image

**Comment by crystalynliu** [comment]
And could you provide the cap for crowdloan as well? @tinydjp

File: crowdloans/projects/kusama.json [hunk 2]
```diff
@@ -338,6 +338,20 @@
...
+    "token": "DORA",
+    "description": "DoraFactory Kusama parachain",
+    "overview": "Dora Factory is the DAO-as-a-Service infrastructure that empowers decentralized organizations and open source ventures. The DORA KSM Parachain is Dora Factory’s Kusama parachain, which supports pallets like quadratic funding and multisig wallet. In connection with DoraHacks, the Dora Factory KSM Parachain will be able to support open source communities, hackathons, quadratic funding grants, and DAOs in general from the Kusama ecosystem.",
+    "website": "https://dorafactory.org",
+    "icon": "https://auction-us-east-2.s3.us-east-2.amazonaws.com/icons/chains/dorafactory.svg",
+    "tokenomics": {
+      "totalSupply": 33000,
+      "data":[
+        { "name": "Crowdloan", "value": 100, "color": "#bdc9ef" } 
+      ]
```
Comment (crystalynliu) [inline crowdloans/projects/kusama.json:354]: Seems the JSON format is no correct

File: crowdloans/rewards/kusama.json [hunk 3]
```diff
@@ -336,5 +336,23 @@
...
+      {
+        "label": "Heiko Additional Bonus",
+        "content": ["4 HKO per KSM contributed"]
+      },
+      {
+        "label": "Vesting schedule",
+        "content": [
+          "100% vests over the lease period of 48 weeks."
+        ]
+      }
```
Comment (crystalynliu) [inline crowdloans/rewards/kusama.json:356]: Seems the JSON format is not correct

**Review by crystalynliu — CHANGES_REQUESTED** [review]

**Comment by tinydjp** [comment]
> Hi @tinydjp could you provide the endblock of the crowdloan?

May I know is it same as the aution details?

**Comment by crystalynliu** [comment]
> > Hi @tinydjp could you provide the endblock of the crowdloan?
> 
> May I know is it same as the aution details?

Yes, it should be your crowdloan details when creating it on relaychain

**Comment by tinydjp** [comment]
> And could you provide the cap for crowdloan as well? @tinydjp

11000KSM

**Comment by tinydjp** [comment]
@crystalynliu could you help review again?

**Review by crystalynliu — APPROVED** [review]

**Comment by crystalynliu** [comment]
@GopherJ could you create vault for Dora team?

ParaId: 2115
Lease period: 21-28
cap: 11000 KSM
endblock: 12197400 (maybe we give a estimate here, we can change it later)

**Comment by GopherJ** [comment]
> 12197400

done

**Pull request merged**